release/create_release_archives.sh

```diff
@@ -69,6 +77,13 @@ mkdir -p "${BIN_DIR}"
 CP=${CP:-"cp"}
 TAR=${TAR:-"tar"}
 
+function replace_with_release_charts_url() {
```

Comment:
it looks like you want do perform this replacement for monthly/LTS/patch release only, but not daily releases, which are stored in the prerelease directory only.

Reply:
Could you point me out which flag can be used to check it is not a daily release? Thanks.

Reply:
@clyang82 you can check if
CB_PIPELINE_TYPE == monthly
you will also need to a line at https://github.com/istio/istio/blob/master/prow/release-test.sh#L47 declaring type as daily
export CB_PIPELINE_TYPE=daily

Reply:
@rkpagadala Thanks for your help. I have updated accordingly.